activerecord/test/cases/time_precision_test.rb

```diff
@@ -2,6 +2,10 @@
 
 if ActiveRecord::Base.connection.supports_datetime_with_precision?
 class TimePrecisionTest < ActiveRecord::TestCase
+  self.use_transactional_fixtures = false
```

Comment:
Can `uses_transaction :test_formatting_time_according_to_precision` to skip the fixtures transaction for that single test.